lib/AssemblyUtil/FastaToAssembly.py

```diff
@@ -186,6 +186,7 @@ def _import_fasta_mass(self, params, max_cumsize=_MAX_DATA_SIZE * _SAFETY_FACTOR
 
         for out, ai in zip(output, assembly_infos):
             out['upa'] = _upa(ai)
+            out['object_info'] = ai
```

Comment:
It kind of boggles my mind a little bit we didn't do this in the first place... it's right there for heck's sake